npm-shrinkwrap.json

```diff
@@ -7866,6 +7866,910 @@
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
       "dev": true
     },
+    "fsevents": {
```

Comment:
fsevent should not be in here

Reply:
Jenkins knows that this is optional and not supported, it will be skipped 👍

Reply:
```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):

npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```
https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/frontend%2Fdcos-ui-oss-pipeline/detail/PR-2838/2/pipeline/46#step-51-log-149

Reply:
Still it should not be added in this PR